### ISSUE TITLE
Laplace approximation for models outputting predictions in 2D

### ISSUE THREAD

**Issue by adeschemps** [AlexImmer/Laplace#104]
Hi, 
first, thanks a lot for this impressive package. 
My goal is object counting in microscopy images, by doing something analoguous to https://www.robots.ox.ac.uk/~vgg/publications/2015/Xie15/weidi15.pdf, basically by integrating density maps predicted over an image containing objects of interest. 

Currently, I am predicting this density map with UNet, and I am getting satisfying results. However, I would like to have some measure of pixel-wise predictive uncertainty, which is how I found your package. My model is trained by minimizing a pixel-wise cross entropy, does it make sense to use Laplace approximations to get pixel-wise predictive uncertainty ? 

From my limited understanding, it should still be possible to compute the FIM by computing the gradients of the model wrt the scalar loss, but I still wanted to ask before I attempt some modifications to try to make this work, as a quick reference search did not yield much results (most people doing uncertainty quantification for image segmentation use some variation of MC dropout).
